extpsdk/pytest.ini

```diff
@@ -5,7 +5,7 @@ filterwarnings =
 junit_logging = all
 junit_suite_name=io.vantiq.extsrc.extpsdk
 minversion = 6.0
-addopts = -rE -q --junit-xml=./build/test-results/test/TEST-io.vantiq.extsrc.extpsdk.pytest.xml
```

Comment:
tired of parsing XML, build html report

Reply:
@fhcarter, does this get generated by our Jenkins job, too?
Do we need to add a step to upload that report to Jenkins?

Reply:
We could do that. It doesn't look like the others, but having it available isn't a bad thing. So far, I just rerun locally and find issues that way, but having it around wouldn't hurt.

Reply:
OK, I added a publishHTML for that report to both Jenkins build definitions.
```
            // publish the generated python-sdk HTML report  (see PR 531)
            publishHTML target: [
                    allowMissing: false,
                    alwaysLinkToLastBuild: false,
                    keepAll: true,
                    reportDir: 'vantiq-extension-sources/extpsdk/build/reports',
                    reportFiles: 'tests/test/index.html',
                    reportTitles: 'ExtPSDK Report',
                    reportName: 'External-Python SDK Report'
            ]
```